tests/pytests/unit/modules/virt/test_domain.py

```diff
@@ -1060,3 +1064,944 @@ def test_update_nic_hostdev_nochange(make_mock_network, make_mock_vm, test):
     define_mock.assert_not_called()
     domain_mock.attachDevice.assert_not_called()
     domain_mock.detachDevice.assert_not_called()
+
+
+def test_update_no_param(make_mock_vm):
+    """
+    Test virt.update(), no parameter passed
+    """
+    domain_mock = make_mock_vm()
+    ret = virt.update("my_vm")
+    assert not ret["definition"]
+    assert not ret.get("mem")
+    assert not ret.get("cpu")
+
+
+def test_update_cpu_and_mem(make_mock_vm):
+    """
+    Test virt.update(), update both cpu and memory
+    """
+    domain_mock = make_mock_vm()
+    ret = virt.update("my_vm", mem=2048, cpu=2)
+    assert ret["definition"]
+    assert ret["mem"]
+    assert ret["cpu"]
+    setxml = ET.fromstring(virt.libvirt.openAuth().defineXML.call_args[0][0])
+    assert "2" == setxml.find("vcpu").text
```

Comment:
For improved diff readability of what pytest reports when there are failures, the asserts should be in the form of:
```python
assert what_we_got == what_we_expect
```

Reply:
The above applies to almost all of the newly added asserts

Reply:
oh, that was never clear to me which way it was... and I think we had both ways depending on the code. I'll fix those today

Reply:
@s0undt3ch I changed the ones introduced in this PR. To ease patches handing, I will change the other asserts in the virt pytests once the opened PRs are merged.

Reply:
Sounds good

Reply:
Thank You.